lib/relay/relay_info_util.dart

```diff
@@ -5,11 +5,15 @@ import 'relay_info.dart';
 class RelayInfoUtil {
   static Future<RelayInfo?> get(String url) async {
     late Uri uri;
-    if (url.indexOf("wss://") == 0) {
+    if (url.startsWith("wss://")) {
       uri = Uri.parse(url).replace(scheme: 'https');
-    } else {
+    } else if (url.startsWith("ws://")) {
       uri = Uri.parse(url).replace(scheme: 'http');
+    } else if (!url.contains("://")) {
```

Comment:
This was causing an error to show in the console when the url parameter did not start with "wss://" or "ws://", so this is a minor bug fix.